kinds/apis/v1beta1/config_types.go

```diff
@@ -145,19 +145,20 @@ type Extensions struct {
 	Helm *Helm `json:"helm,omitempty"`
 }
 
+type Domains struct {
+	ReplicatedAppDomain string `json:"replicatedAppDomain,omitempty"` // Replicated App Domain
```

Comment:
Any reason not to make this an endpoint? Maybe it is better to call it ReplicatedAPIEndpoint to make it more generic.

Reply:
I don't have strong feelings either way

Reply:
Per a discussion irl we will keep this as is.

Reply:
Had debate in standup, keeping 'ReplicatedAppDomain'